### PR TITLE
CBUFFER support on GLCore and ES3 [DO NOT MERGE]

### PR DESCRIPTION
**Pull request by aleks01010101**
### Purpose of this PR
Enable CBUFFER for GLCore and GLES3.

---
### Release Notes
OpenGL Core and OpenGL ES 3 now use CBUFFERs.

---
### Testing status
**Katana Tests**: No tests run GL anyway.

**Manual Tests**: Editor + device testing + QA.

**Automated Tests**: Nothing

Any test projects to go with this to help reviewers?

---
### Overall Product Risks
**Technical Risk**: None, Low, Medium, High?
None
**Halo Effect**: None, Low, Medium, High?
Medium
---
### Comments to reviewers
Notes for the reviewers you have assigned.

### CODE REVIEW

File: com.unity.render-pipelines.core/CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [5.3.0] - 2018-11-xx
+### Changed
+- Enabled CBUFFER for GLCore and GLES3.
```
Comment (Katstodian) [inline com.unity.render-pipelines.core/CHANGELOG.md]: Should this be under Added instead?

**Comment by aleks01010101** [comment]
I'll update the section when it's time to merge it.

File: com.unity.render-pipelines.core/CHANGELOG.md (hunk 1 above)
Comment (aleks01010101) [reply com.unity.render-pipelines.core/CHANGELOG.md]: Well, from my perspective, this is a change (CBUFFERS were disabled and now are enabled). It's not like there's a big change on surface anyway :)

Comment (Katstodian) [reply com.unity.render-pipelines.core/CHANGELOG.md]: Sold! Sorry for the late approve. :)

**Comment by sebastienlagarde** [comment]
Sorry, made a mistake and merged this PR, I have reverted it. 
Note: Usually when we merge and unmerge a PR like this, it is sometime not recognize by git at the next merge, you may have to create a new PR, sorry.

**Pull request merged**